### PR TITLE
Feature/docassembly skeleton project setup

### PR DESCRIPTION
**Pull request by benouaer**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/dto/AbstractAuditingDTO.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.dg.docassembly.service.dto;
+
+import java.io.Serializable;
+import java.time.Instant;
+
+/**
+ * Base abstract class for DTO which will hold definitions for created, last modified by and created,
+ * last modified by date.
+ */
+public abstract class AbstractAuditingDTO implements Serializable {
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/dto/AbstractAuditingDTO.java]: @benouaer you won't need this stufff

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+app-insights:
+  dev-mode: off
+  request-component: on
+  telemetry-component: on
+
+em-rpa-ccd-orchestrator:
+  base-url: ${EM_CCD_ORCHESTRATOR_URL:http://localhost:4623}
+
+dm-store-app:
```
Comment (linusnorton) [inline src/main/resources/application.yaml]: you won't need dm-store

Comment (benouaer) [reply src/main/resources/application.yaml]: I thought I removed it, and I can also see a reference to the orchestrator :S

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/dto/AbstractAuditingDTO.java (hunk 1 above)
Comment (benouaer) [reply src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/dto/AbstractAuditingDTO.java]: we don't need it for auditing?

Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/dto/AbstractAuditingDTO.java]: I didn't think you had a db

File: src/main/resources/application.yaml (hunk 2 above)
Comment (benouaer) [reply src/main/resources/application.yaml]: removed

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/dto/AbstractAuditingDTO.java (hunk 1 above)
Comment (benouaer) [reply src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/dto/AbstractAuditingDTO.java]: removed

**Pull request merged**